unison-src/transcripts-using-base/random-deserial.md

```diff
@@ -0,0 +1,57 @@
+```ucm:hide
+.> builtins.mergeio
+```
+
+```unison
+directory = "unison-src/transcripts-using-base/serialized-cases/"
+
+availableCases : '{IO,Exception} [Text]
+availableCases _ =
+  l = filter (contains ".ser") (directoryContents directory)
+  map (t -> Text.take (drop (Text.size t) 4) t) l
+
+gen : Nat -> Nat -> (Nat, Nat)
+gen seed k =
+  c = 1442695040888963407
```

Comment:
Chosen by fair dice roll, guaranteed to be random? 😋

Reply:
I just copied it out of base. No idea how Paul (I think) chose it. :)